src/Microsoft.AspNet.Authentication.JwtBearer/Events/JwtBearerEvents.cs

```diff
@@ -23,22 +23,22 @@ public class JwtBearerEvents : IJwtBearerEvents
         /// <summary>
         /// Invoked when a protocol message is first received.
         /// </summary>
-        public Func<MessageReceivedContext, Task> OnMessageReceived { get; set; } = context => Task.FromResult(0);
+        public Func<ReceivingTokenContext, Task> OnMessageReceived { get; set; } = context => Task.FromResult(0);
```

Comment:
Another reminder of why I wish we only had either the delegate or the function :)